tools/go-cli/go-whisk-cli/main.go

```diff
@@ -74,14 +76,13 @@ func main() {
 
         // If the err msg should be displayed to the console and it has not already been
         // displayed, display it now.
-        var errMsgPrefix string = "error: "
-        if (exitCode == 0) {
-            errMsgPrefix = ""
-        }
-        if displayMsg && !msgDisplayed {
-            fmt.Fprintf(os.Stderr, "%s%s\n", errMsgPrefix, err)
+        if displayMsg && !msgDisplayed && exitCode != 0 {
+            fmt.Fprintf(os.Stderr, "%s%s\n", color.RedString("error: "), err)
+        } else if displayMsg && !msgDisplayed && exitCode != 0 {
```

Comment:
Comparing conditional `if` expressions on lines 79 and 81.  Same expression?  Perhaps line 81 is mean to have `exitCode == 0`